aiida/storage/sqlite_temp/backend.py

```diff
@@ -120,12 +127,13 @@ def get_session(self) -> Session:
             self._session.commit()
         return self._session
 
-    def get_repository(self) -> SandboxRepositoryBackend:
+    def get_repository(self) -> SandboxShaRepositoryBackend:
         if self._closed:
             raise ClosedStorage(str(self))
         if self._repo is None:
             # to-do this does not seem to be removing the folder on garbage collection?
-            self._repo = SandboxRepositoryBackend(filepath=get_config_option('storage.sandbox') or None)
+            repo_path = self.profile.storage_config.get('repo_path')
+            self._repo = SandboxShaRepositoryBackend(filepath=Path(repo_path) if repo_path else None)
```

Comment:
If `repo_path` can be unset and you can accept `None`, wouldn't it be better to do:
```suggestion
            repo_path = self.profile.storage_config.get('repo_path', None)
            self._repo = SandboxShaRepositoryBackend(filepath=Path(repo_path) if repo_path else None)
```
Otherwise this could except